pages/events/[event].tsx

```diff
@@ -27,12 +37,12 @@ const EventPage: NextPage = () => {
   const { supabase, user } = useSupabase();
   const [dataFetchErrors, setDataFetchErrors] = useState<string[]>([]);
   const [eventData, setEventData] = useState<Event | null>();
-
-  const [countdown, setCountdown] = useState(10);
+  const [dateStatus, setDateStatus] = useState<DateStatus>(DateStatus.loading);
```

Comment:
Does this need to be a state? I'm pretty sure we can just recompute it on every render (it's very cheap) and that makes the flow a little easier to follow IMO